server/controller/src/main/java/ai/starwhale/mlops/domain/job/ModelServingService.java

```diff
@@ -126,16 +140,23 @@ public ModelServingVo create(
                 .modelVersionId(model.getId())
                 .jobStatus(JobStatus.CREATED)
                 .resourcePool(resourcePool)
+                .lastVisitTime(new Date())
                 .build();
 
-        modelServingMapper.add(entity);
+        long id;
+        synchronized (this) {
```

Comment:
Is the synchronized block written to ensure idempotence? if insert is ignore for exception, it seems unnecessary to use sync block

Reply:
This block is for preventing the race condition with the GC thread, but the synchronized range in the GC thread is wrong, I will change it

Reply:
Done in 8f65c60